.github/workflows/main.yml

```diff
@@ -18,7 +18,6 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 2.4
-          bundler-cache: true
```

Comment:
It seems like this line was the reason why the build for Ruby 2.4 and 2.5 was failing.